### DISCUSSION TITLE
Setting Starting Values for Fitting

### DISCUSSION THREAD

**Discussion by bliblabl** [georgebv/pyextremes#59] [Ideas]
I have a lot of appreciation for you work and really enjoy using it - thanks!

I had to make a scipy.stats.rv_continuous method myself and to fit it I need to set the starting values manually (arg1, arg2, arg3 from here: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.rv_continuous.fit.html#scipy.stats.rv_continuous.fit). Is it already possible to do this via fit_model? If not, would it be possible to integrate this in pyextremes?

Would be really helpfull!

**Comment by georgebv** [comment]
It is not currently possible and I am not sure what would be the use case for this. This would only be applicable to the `MLE` model because `Emcee` doesn't even use scipy's `fit` method.

Regarding your problem, I think you can hardcode those in the `_fitstart(data)` method as specified in the scipy docs:

```python
def _fitstart(self, data):
    return arg1, arg2, arg3
```

where `argx` is a hardcoded value

**Reply by bliblabl** [reply]
Thanks a lot. I did that and it worked (before I overloaded the _fit() method and that would have worked as well for the MLE).
